Source/CronosPlayUnreal/Public/DefiWalletCoreActor.h

```diff
@@ -847,17 +820,14 @@ class CRONOSPLAYUNREAL_API ADefiWalletCoreActor : public AActor {
      * @param contractAddress erc20 contract
      * @param erc20owner erc20 owner
      * @param erc20spender erc20 spender
-     * @param result allowance
-     * @param success whether succeed or not
-     * @param output_message error message, "" if succeed
+     * @param Out callback
      */
 
     UFUNCTION(BlueprintCallable,
               meta = (DisplayName = "Erc20Allowance", Keywords = "Wallet"),
               Category = "CronosPlayUnreal")
     void Erc20Allowance(FString contractAddress, FString erc20owner,
-                        FString erc20spender, FString &result, bool &success,
-                        FString &output_message);
+                        FString erc20spender, FGeneralCronosCallDelegate Out);
```

Comment:
Same, better off assigning a dedicated delegate. Rename it.

Reply:
ok